modules/serverless-logic-config-persistence-using-sonataflowplaform-cr.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-persistence
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-config-persistence-using-sonataflowplaform-cr_{context}"]
+= Configuring persistence using the `SonataFlowPlatform` CR
+
+The `SonataFlowPlatform` CR enables persistence configuration at the namespace level. This approach applies the persistence settings automatically to all workflows deployed in the namespace. It simplifies resource configuration, especially when multiple workflows in the namespace belong to the same application. While this configuration is applied by default, individual workflows in the namespace can override it using the SonataFlow CR.
+
+The {ServerlessLogicOperatorName} also uses this configuration to set up persistence for supporting services. 
+
+[NOTE]
+====
+The persistence configurations are applied only at the time of workflow deployment. Changes to the `SonataFlowPlatform` CR do not affect workflows that are already deployed.
+====
+
+You can configure persistence in the `SonataFlowPlatform` CR by using the following procedure:
+
+* Define the `SonataFlowPlatform` CR.
++
+Specify the persistence settings in the `persistence` field under the `SonataFlowPlatform` CR spec.
++
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform-example
+  namespace: example-namespace
+spec:
+  persistence:
+    postgresql:
+      serviceRef:
+        name: postgres-example <1>
+        namespace: postgres-example-namespace <2>
+        databaseName: example-database <3>
+        port: 1234 <4>
+      secretRef:
+        name: postgres-secrets-example <5>
+        userKey: POSTGRESQL_USER <6>
+        passwordKey: POSTGRESQL_PASSWORD <7>
+----
+<1> Name of the Kubernetes Service connecting to the PostgreSQL database.
+<2> Optional: Namespace of the PostgreSQL Service. Defaults to the namespace of the `SonataFlowPlatform`.
+<3> Name of the PostgreSQL database for storing workflow data.
+<4> Optional: Port number to connect to the PostgreSQL service. Defaults to `5432`.
+<5> Name of the Kubernetes Secret containing database credentials.
+<6> Key in the Secret that contains the database username.
+<7> Key in the Secret that contains the database password.
+
+When this persistence configuration is in place, the {ServerlessLogicOperatorName} configures every workflow deployed in this namespace using the `preview` or `gitops` profile, to connect with the PostgreSQL database by injecting relevant JDBC connection parameters as environment variables. 
```

Comment:
This is not something that really needs to be addressed in this PR, however in this module, this is a lot of conceptual information to add after a procedure's steps. I feel like the spirit of procedural modules are to focus on instructions for a task, and talking about conceptual content in depth should be saved for concept modules.

I'd suggest considering a structure like this for the future:

* Conceptual module that introduces the topic
  * Procedural module that just focuses on the steps